src/android/AdvancedAudioDevice.java

```diff
@@ -0,0 +1,1175 @@
+package com.tokbox.cordova;
```

Comment:
Added custom audio driver class. In this, we have replaced STREAM_VOICE_CALL with STREAM_MUSIC in order to enhance the audio output. 
I have put a DEV document whereever we have made this change for future reference.